### ISSUE TITLE
Can't use for Windows 10

### ISSUE THREAD

**Issue by ligun** [jinput/jinput#16]
I used jinput 2.0.6 to input game pad.
I installed this library with the Gradle.

```
    compile 'net.java.jinput:jinput:2.0.6'
    compile 'net.java.jinput:jinput-platform:2.0.6'
```

But it could not run with the Windows 10.
It's outputed error message.

```
Found unknown Windows version: Windows 10
```

I think that this issue have already resolved for 643e638
But maven repo is not updated yet.
So please update maven repo.

**Comment by Endolf** [comment]
I've placed a release in the staging area of maven central, can you confirm that this works please before I release it.

Add the following to your pom to test.
```
    <repositories>
        <repository>
            <id>staging</id>
            <url>https://oss.sonatype.org/content/repositories/netjavajinput-1002/</url>
        </repository>
    </repositories>
```

Thanks

**Comment by Endolf** [comment]
I've released 2.0.7, closing.

**Comment by ligun** [comment]
Thank you for your response.
I tried new version and the result was very good.
I have no problem to use gamepad.
